hw_classes/myrna.py

```diff
@@ -0,0 +1,29 @@
+from Bio.Seq import Seq
+
+
+class MyRNA:
+
+    """A class that describes RNA,
+    with traslation method and reverse transcription method"""
+
+    def __init__(self, sequence=''):
+        rna_alphabet = ['A'.casefold(), 'U'.casefold(), 'G'.casefold(), 'C'.casefold()]

```

Comment:
Когда мы часто проверяем вхождение в какое-то множество, то лучши использовать эти самые множества. Они работают чуть быстрее
```suggestion
        rna_alphabet = {'A'.casefold(), 'U'.casefold(), 'G'.casefold(), 'C'.casefold()}

```